### PR TITLE
adding a version eye page

### PR DESCRIPTION
**Pull request by tkadlec**

### CODE REVIEW

File: _features_versioneye/01_integrate.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+# Page title.
+title: Broad Ecosystem Support
+
+# Image should be served via https.
+image: https://res.cloudinary.com/snyk/image/upload/c_scale,q_auto,w_550/v1513193820/vulndb.png
+
+# Used as the alt text.
+image-caption: "A screenshot of the Snyk website showing test reports alongside each project"
```
Comment (geva) [inline _features_versioneye/01_integrate.md]: Please use an image that shows more high severity vulns than the one currently in use. Also make sure your .net and PHP flags are set so the vuln db filters on top show all the ecosystems

File: _features_versioneye/04_fixes.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+# Page title.
+title: Automated Fixes
```
Comment (geva) [inline _features_versioneye/04_fixes.md]: I would move this after 'Broad Ecosystem Support'

**Review by geva — APPROVED** [review]

File: _features_versioneye/04_fixes.md (hunk 2 above)
Comment (tkadlec) [reply _features_versioneye/04_fixes.md]: done

File: _features_versioneye/01_integrate.md (hunk 1 above)
Comment (tkadlec) [reply _features_versioneye/01_integrate.md]: done

**Pull request merged**